src/utilities/gql.ts

```diff
@@ -2521,6 +2604,38 @@ const gql = {
     }
   `,
 
+  SUB_MOST_RECENT_EXPANSION_FOR_SIMULATION_SEQS: `#graphql
+    subscription SubMostRecentExpansion {
+      expanded_sequences {
```

Comment:
Do you mind using the existing `Queries` enums for these subscription queries?

Reply:
I believe this comment has been addressed, so I'll resolve it. I'm going to read this code a bit more carefully and make a new, unrelated comment.